src/multiStepInput.ts

```diff
@@ -71,6 +70,7 @@ export class MultiStepInput {
 		}
 	}
 
+	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
did you consider turning these rules off in eslintrc?

Reply:
I had thought about it, but do think these rules would help make the code a bit friendlier if we commit to fixing these as we go along. If it ends up being a wontfix, can add new rules to the eslintrc :)